product_docs/docs/pgd/4/upgrades/upgrade_paths.mdx

```diff
@@ -4,31 +4,31 @@ title: Supported BDR upgrade paths
 
 ## Upgrading within version 4
 
-Beginning with version 4, EDB Postgres Distributed has adopted semantic versioning. All changes within the same major will be backward compatible lowering the risk when upgrading and allowing you to choose any later minor or patch release as the upgrade target. 
+Beginning with version 4, EDB Postgres Distributed has adopted [semantic versioning](https://semver.org/). All changes within the same major version will be backward compatible, lowering the risk when upgrading and allowing you to choose any later minor or patch release as the upgrade target. You can upgrade from any version 4.x release to a later 4.x release.
 
-| 4.0.0 | 4.0.1 | 4.0.2 | 4.1.0 | 4.1.1 | Target BDR version |
-|-------|-------|-------|--------|------|--------------------|
-| ✓     | ✓     | ✓     | ✓      | ✓    | 4.2.0              |
-| ✓     | ✓     | ✓     | ✓      |      | 4.1.1              |
-| ✓     | ✓     | ✓     |        |      | 4.1.0              |
-| ✓     | ✓     |       |        |      | 4.0.2              |
-| ✓     |       |       |        |      | 4.0.1              |
 
 ## Upgrading from version 3.7 to version 4
 
-Currently it is recommended that you are using 3.7.15 or later before upgrading to 4. See [Upgrading within from 3.7](/pgd/3.7/bdr/upgrades/supported_paths/#upgrading-within-version-37) in the 3.7 documentation for more information.  After upgrading to 3.7.15 or later the following combinations are allowed
+Generally, we recommend you upgrade to the latest version 3.7 release, before
+upgrading to the latest version 4 release. You will have to be using 3.7.15 or later 
+before upgrading to 4. See [Upgrading within from 3.7](/pgd/3.7/bdr/upgrades/supported_paths/#upgrading-within-version-37) in the
+3.7 documentation for more information. After upgrading to 3.7.15 or later, the
+following upgrade paths are possible.
+
+| From version | To version |
+| ---- | -- |
+| 3.7.15 | 4.0.2 or later |
+| 3.7.16 | 4.1.0 or later |
+| 3.7.17 | 4.1.1 or later |
+| 3.7.18 | 4.2.1 or later |
+| 3.7.19 | 4.2.2 or later |
+| 3.7.20 | 4.3.0 or later |
+| 3.7.21 | 4.3.1 or later |
+| 3.7.22 | 4.3.2 or later |
```

Comment:
I checked the release tags specifically and figured this table matches more the "has been released at the same time" rather than what the upgrade capabilities actually are. The "possible upgrade paths" deviate slightly and should look more like:

To me, it looks more like:
```
| From version | To version |
| ---- | -- |
| 3.7.15 | 4.0.2 or later |
| 3.7.16 | 4.1.0 or later |
| 3.7.17 | needs to be upgraded to a newer 3.7.x version, first |
| 3.7.18 | 4.2.1 or later |
| 3.7.19 | 4.2.2 or later |
| 3.7.20 | 4.3.0 or later |
| 3.7.21 | 4.3.1 or later |
| 3.7.22 | 4.3.2 or later |
```

Reply:
Switched to the given list